modules/application/src/main/java/edu/northeastern/truthtree/controller/basicInfo/BasicInfo.java

```diff
@@ -48,11 +51,10 @@ public BasicInfo(IBasicInfoService service) {
    */
   @Override
   @RequestMapping(value = "/api/states", method = RequestMethod.GET)
-  public JSONArray getBasicStatesInfo(@RequestParam
-      (value = POPULATION_RANGE, required = false)
-      int[] range) {
-
-    return this.service.getBasicStatesInfo(range);
+  public ResponseEntity<PageDTO<LocationDTO>> getBasicStatesInfo(
+          @RequestParam(value = POPULATION_RANGE, required = false) int[] range,
+          @RequestParam(name = "page", defaultValue = "1") int page) throws IOException {
```

Comment:
Can you rename page to pageNumber? I guess other pagination supported apis have used pageNumber.

Reply:
addressed